src/language-css/utils.js

```diff
@@ -211,6 +211,14 @@ function hasParensAroundValueNode(node) {
   );
 }
 
+function isPostcssSimpleVar(currentNode, nextNode) {
+  return (
+    currentNode.type === "value-func" &&
```

Comment:
`currentNode.name === '$$'`

Reply:
/cc @ad1992

Reply:
@evilebottnawi if I add the check  `currentNode.value === '$$'` then it will only work for functions with name `$$`, etc. Is that the correct behaviour ? Then I should also add test where function name is something else other than `$$`

Reply:
@ad1992 postcss simple var have `$$(name)` syntax and it is normal, yes add test where we have `@@(name)`, maybe other tests too

Reply:
got it. Made the changes.